health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/viewVillage.js

```
@@ -140,6 +141,30 @@ const VillageView = () => {
   }, [employeeData]);
 
+  // fetch the process instance for the current microplan to check if we need to disabled actions or not
+  const { isLoading: isProcessLoading, data: processData, } = Digit.Hooks.useCustomAPIHook({
+    url: "/egov-workflow-v2/egov-wf/process/_search",
+    params: {
+      tenantId: tenantId,
+      history: true,
+      businessIds: microplanId,
+    },
+    config: {
+        cacheTime:Infinity,
```

Biome

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)